modules/custom-status/custom-status.php

```diff
@@ -30,10 +30,16 @@ class Custom_Status extends Module {
 	private $custom_statuses_cache = [];
 
 	// This is taxonomy name used to store all our custom statuses
-	const TAXONOMY_KEY = 'post_status';
+	const TAXONOMY_KEY = 'vw_post_status';
```

Comment:
Changed the taxonomy so it won't interfere with Edit Flow. I left the slug as is as that would create more problems imo, since we have some statuses that cannot be deleted and their slug matches core slugs. We would need to add in branching logic to tackle that. This keeps it a bit simpler.